hw03_babayemie.py

```diff
@@ -0,0 +1,164 @@
+#################################################################################
+# Author: Elijah Babayemi
+# Username: babayemie
+#
+# Assignment:HW03: Fully Functional Gitty Psychedelic Robotic Turtles
+# Purpose: Continue practicing creating and using functions, more practice on using the turtle library, learn about how computers represent colors, and learn about source control and Git.
+#
+# Google Doc Link: https://docs.google.com/document/d/1X9d90s3RxZa3LG8GFm7Y8XtmVSOv1ai_OpNtu0xGtpk/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+
+
+import turtle
+
+
+
+
+def make_house():
+   """
+  Draws the main body of the house
+   """
+   turtle.penup()
+   turtle.goto(-50, -50)
+   turtle.pendown()
+
+
+   turtle.fillcolor("#FF3C00")  # red color
+   turtle.begin_fill()
+   for _ in range(4):
```

Comment:
This is a unique way of implementing a `for` loop. You should cite your sources when you look things like this up or ask a friend, to avoid any issues of academic dishonesty...

Reply:
The Acknowledgements section of the header block is where I expect citations